Remove incorrect assertion that may be hit in packages that
set conf_files with files not installed.

Temporary fix until xbps-0.19 is released.

--- lib/package_remove_obsoletes.c.orig	2012-12-03 17:27:30.354154688 +0100
+++ lib/package_remove_obsoletes.c	2012-12-03 17:27:49.560683084 +0100
@@ -67,7 +67,8 @@ again:
 		found = false;
 		obj = prop_array_get(array, i);
 		oldstr = prop_dictionary_get(obj, "file");
-		assert(oldstr);
+		if (oldstr == NULL)
+			continue;
 
 		file = xbps_xasprintf(".%s",
 		    prop_string_cstring_nocopy(oldstr));
